sci-biology/ABI_connectivity_data/ABI_connectivity_data-0.1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit check-reqs
+
+DESCRIPTION="Collection of connectivity data form the Allen Mouse Brain data portal"
+HOMEPAGE="https://github.com/IBT-FMI/ABI_connectivity_data"
+SRC_URI="
+	http://chymera.eu/distfiles/${P}.tar.xz
```

Comment:
one line is enough at this stage, break this up accross lines only when you need to.